javarun/__main__.py

```diff
@@ -0,0 +1,3 @@
+import javarun
+if __name__ == '__main__':
+    javarun.jrun.jrun_main()
```

Comment:
Would just be `javarun.jrun_main` if `jrun_main` is imported in `__init__`.
```suggestion
    javarun.jrun_main()
```